zpu/zpu.c

```diff
@@ -1,89 +1,94 @@
-/* zpu plugin by saucec0de at 2016 */
+// SPDX-FileCopyrightText: 2016 saucec0de
+// SPDX-License-Identifier: LGPL-3.0-only
 
-#include <r_asm.h>
-#include <r_lib.h>
+// zpu plugin by saucec0de at 2016
```

Comment:
No need for this line, since it's already in SPDX

Reply:
Alright, done.